.circleci/unittest/linux/scripts/install.sh

```diff
@@ -23,10 +23,13 @@ else
     version="$(python -c "print('.'.join(\"${CUDA_VERSION}\".split('.')[:2]))")"
 
     cuda_toolkit_pckg="cudatoolkit"
+    cudatoolkit=""
     if [[ "$CU_VERSION" == cu116 ]]; then
         cuda_toolkit_pckg="cuda"
+        cudatoolkit="-c nvidia cuda=11.6 atalman::pytorch-cuda=1.0=cuda11.6"
```

Comment:
Your personal username appears here. Is that accidental or perhaps at temporary workaround?

Reply:
@datumbox This is testing of this package: https://anaconda.org/atalman/pytorch-cuda/files . Here is the PR that introduces this metapackage https://github.com/pytorch/builder/pull/1094

Reply:
Gotcha, I understand it will be moved to pytorch once you are done. Thanks for clarifying.